examples/asyncio/audit/audit_examples/log.py

```diff
@@ -8,6 +8,7 @@
 
 token = os.getenv("PANGEA_AUDIT_TOKEN")
 domain = os.getenv("PANGEA_DOMAIN")
+assert domain
```

Comment:
shouldn't we assert token too?

Reply:
Technically already caught by `ServiceBase()` `__init__()`, but sure I'll add them.